app/src/main/res/xml/preference.xml

```diff
@@ -144,6 +144,12 @@
             app:key="biometric_auth"
             app:summary="@string/biometric_auth_summary"
             app:title="@string/biometric_auth_title" />
+        <EditTextPreference
```

Comment:
I thought about it some more and I've decided I don't wanna expose this as a preference, or even have a timeout. Reauthenticating on each resume feels reasonable.